docs/book/src/cronjob-tutorial/testdata/project/README.md

```diff
@@ -12,7 +12,7 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 1. Install Instances of Custom Resources:
 
 ```sh
-kubectl apply -f config/samples/
+kubectl apply -k config/samples/
```

Comment:
It actually should be make install. See the quick start. We should stick with the recommendations and guidances provided there. 
```suggestion
make install
```

Reply:
@Sajiyah-Salat,

When you finish the task that you are working on .. move forward with this PR in the right way might be a good issue for you so that you start to learn and check how the bollerplates work.

Reply:
@camilamacedo86 Sure. I would do that. If possible need your reviews on the pr I am working on to move forward.